exm/contract.js

```diff
@@ -1,9 +1,41 @@
-export default function handle(state, action) {
-  const { functionRole, title, url } = action.input;
+export function handle(state, action) {
+
+  /**
+   *****POST FORMAT*****
+     const post = {
+      postID: '',
+      author: '',
+      title: '',
+      url: '',
+      description: '',
+      dateAndTime: '',
+      comments: [
```

Comment:
This signature is wrong. The best practice here is a node-based signature

```
interface comments { 
  author: string;
  text: string;
  replies: comments[]
}
```
where the replies is an array of itself.